### PR TITLE
update dataspace template executable info

### PR DESCRIPTION
**Pull request by YannanGao-gs**
#### What does this PR do / why is it needed ?

update dataspace template executable info

#### Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: legend-engine-xts-data-space/legend-engine-xt-data-space-generation/src/main/java/org/finos/legend/engine/generation/analytics/model/DataSpaceTemplateExecutablePointerInfo.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.engine.generation.analytics.model;
+
+public class DataSpaceTemplateExecutablePointerInfo extends DataSpaceExecutableInfo
+{
+    public String id;
+    public String executionContextKey;
+    public String function;
```
Comment (MauricioUyaguari) [inline legend-engine-xts-data-space/legend-engine-xt-data-space-generation/src/main/java/org/finos/legend/engine/generation/analytics/model/DataSpaceTemplateExecutablePointerInfo.java:21]: could we rename this to `executable` or packagablePointer. 
We could add support for other elements (service etc) in the future.

Comment (YannanGao-gs) [reply legend-engine-xts-data-space/legend-engine-xt-data-space-generation/src/main/java/org/finos/legend/engine/generation/analytics/model/DataSpaceTemplateExecutablePointerInfo.java:21]: sounds good will do. But won't it overlap with the existing dataspace executables that already support service?

**Comment by finos-admin** [comment]
This PR is stale because it has been open for 30 days with no activity. Please remove stale label or add any comment to keep this open. Otherwise this will be closed in 5 days.

**Comment by finos-admin** [comment]
This PR was closed because it has been inactive for 35 days. Please re-open if this PR is still relevant.

**Pull request closed without merging**